README.md

```diff
@@ -1,2 +1,62 @@
 # ssm-put-parameter
 Composite GitHub action to put parameters to AWS Parameter Store
+1) Validates input parameters (if each parameter in list of possible values)
+2) Runs `aws ssm put-parameter` command with the parameters
+
+### Parameters
+
+| name | required | possible values | default value |
+|------|----------|-----------------|-----------------|
+| name | true | any | none |
+| value | true | any | none |
+| description | false | any | "" |
+| type | false | ["String", "SecureString", "StringList"] | "String" |
+| overwrite | false | ["overwrite", "no-overwrite"] | "no-overwrite" |
+| tier | false | ["Standard" "Advanced", "Intelligent-Tiering"] | "Standard" |
+ 
+### How to use
+Before running aws cli should be set up. Can be used many times in the same job on 'step' level. 
+
+e.g.:
+```yaml
```

Comment:
Would be nice to make this more general/less contextual. "risks contrived" doesn't mean much to most people in LT :-)